.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Please feel free to remove whatever sections/lines in this aren't relevant. -->
+
+## Title Line Template: [Website] - [UI components]: [Brief description.]
```

Comment:
Changed to singular "component" here e11f14c

@maya The distinction between "Website" and "UI component" was implemented because the website code and the UI component code live within the same repo. It's important for us to be able to have a clear idea which one a particular issue is addressing so that we can better assist our contributors.